tools/cheribsdbox/Makefile

```diff
@@ -295,15 +287,9 @@ CRUNCH_ALIAS_w=	uptime
 CRUNCH_PROGS_usr.bin+=su
 
 # Tracing currently only implemented for CHERI-QEMU architectures
```

Comment:
Not quite true, CHERI-QEMU supports Morello but we don't have a magic tracing hint for it